dist_config.py

```diff
@@ -1,21 +1,21 @@
 # -*- coding: utf-8 -*-
 
 # Cython version to cythonize the code.
-CYTHON_VERSION = '0.28.3'
+CYTHON_VERSION = '0.29.21'
 
 # Key-value of sdist build settings.
 # See descriptions of WHEEL_LINUX_CONFIGS for details.
 # Note that cuDNN and NCCL must be available for sdist.
 SDIST_CONFIG = {
-    'image': 'nvidia/cuda:9.0-cudnn7-devel-centos7',
+    'image': 'nvidia/cuda:9.2-cudnn7-devel-centos6',
     'nccl': {
         'type': 'v2-tar',
         'files': [
             'nccl_2.5.6-1+cuda9.0_x86_64.txz',
```

Comment:
Is it ok to use this nccl version for cuda9.0 instead of 9.2?
I guess they are binary compatible

Reply:
Thanks, it should not affect the result (as this is only sdist (generate cpp from pyx)), it's better to use cuda 9.2 version.